src/app/code/community/Smile/ElasticSearch/Model/Resource/Engine/Elasticsearch/Index.php

```diff
@@ -521,17 +539,15 @@ public function loadMappingPropertiesFromIndex($type)
     /**
      * Create document to index.
      *
-     * @param string $id   Document Id
-     * @param array  $data Data indexed
-     * @param string $type Document type
```

Comment:
In another one you've @deprecated it rather than removing the line. Not sure what's best here.

Reply:
My mistake here. It actually is used at some location for special document types (`stats`). So I reintroduced the param but forgot to update the docblock.